packages/launcher-client/src/client/impl/default.launcher.client.ts

```diff
@@ -204,6 +206,12 @@ export default class DefaultLauncherClient implements LauncherClient {
     if (config.clusterId) {
       headers['X-OpenShift-Cluster'] = config.clusterId;
     }
+    if (config.clusterUrl) {
+      headers['X-OpenShift-Cluster-URL'] = config.clusterUrl;
+    }
+    if (config.clusterToken) {
+      headers['X-OpenShift-Authorization'] = config.clusterUrl;
```

Comment:
This looks wrong to me. It should always be a JWT token

Reply:
good catch, that was definitely wrong